src/App.tsx

```diff
@@ -1,10 +1,48 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { wait } from './utils/fetchClient';
+
+enum SortFild {
+  All = 'all',
+  Completed = 'completed',
+  Active = 'active',
+}
```

Comment:
It would be better to move enum to a separate type

Reply:
> It would be better to move enum to a separate type

Good point, and fix typo